apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -236,17 +299,11 @@ type ClusterResourcePlacementStatus struct {
 	// +optional
 	SelectedResources []ResourceIdentifier `json:"selectedResources,omitempty"`
 
-	// TargetClusters contains a list of names of member clusters selected by PlacementPolicy.
+	// PlacementStatuses contains a list of placement status on the clusters that are selected by PlacementPolicy.
+	// Each selected cluster is guaranteed to have a corresponding placementStatuses.
 	// Note that the clusters must be both joined and meeting PlacementPolicy.
```

Comment:
The clusters may not meet placement policy as of now. Please update. They only meet policy when they were first selected.

Reply:
> The clusters may not meet placement policy as of now

I am not sure what this means